sale_stock_expiry_date_on_qty_at_date_widget/views/sale_order_line_view.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2025 Moduon Team S.L. <info@moduon.team>
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+<odoo>
+    <record id="view_order_form_inherit_sale_stock_qty" model="ir.ui.view">
+            <field
+            name="name"
+        >sale.order.line.tree.sale.stock.predicted.expiration.date</field>
+            <field
+            name="inherit_id"
+            ref="sale_stock.view_order_form_inherit_sale_stock_qty"
+        />
+            <field name="model">sale.order</field>
+            <field name="arch" type="xml">
+                <xpath
+                expr="//page/field[@name='order_line']/form//field[@name='company_id']"
+                position="after"
+            >
+                    <!-- below fields are used in the widget qty_at_date_widget -->
+                    <field name="predicted_first_expiration_date" invisible="1" />
+                </xpath>
+                <xpath
+                expr="//page/field[@name='order_line']/tree//field[@name='company_id']"
+                position="after"
+            >
+                    <!-- below fields are used in the widget qty_at_date_widget -->
+                    <field name="predicted_first_expiration_date" invisible="1" />
+                </xpath>
+                <xpath
+                expr="//page/field[@name='order_line']/kanban//field[@name='company_id']"
+                position="after"
+            >
+                    <!-- below fields are used in the widget qty_at_date_widget -->
+                    <field name="predicted_first_expiration_date" invisible="1" />
+                </xpath>
+            </field>
+        </record>
```

Comment:
suggestion: indentation looks a bit weird in this file. Prettier is whitespace-conservative, so its reformats are sometimes not what you'd expect. Would you mind to prettify it a bit please?